skills/skill-alertmanager/__init__.py

```diff
@@ -18,16 +18,17 @@ async def alertmanager(self, event: Request):
                       pprint.pformat(payload))
 
         for alert in payload["alerts"]:
-            msg = ""
+            template = "New alert: {} in {}"
```

Comment:
I think we should keep the message as it is and only change the template below, including the origin.